numpy/core/src/multiarray/datetime.c

```diff
@@ -596,9 +596,9 @@ convert_datetime_to_datetimestruct(PyArray_DatetimeMetaData *meta,
                 out->as = (int)((dt % 1000LL) * 1000);
             }
             else {
-                npy_datetime minutes;
+                int minutes;
```

Comment:
Needs to be `int` for the call?

Reply:
Yep, and we're dividing by such a big number that that seems pretty reasonable